File: vod_content_rating_frontend/print_android_root.py
#!/usr/bin/env python3
import json, os
root = os.path.join('content-rating-display-213568', 'vod_content_rating_frontend')
modules = [os.path.join(root, m) for m in ('app','list','utilities')]
print(json.dumps({
    "android": {
        "gradle_root": root,
        "modules": modules,
        "instructions": [f"cd {root}", "./gradlew build"]
    }
}, indent=2))

File: vod_content_rating_frontend/print-android-root.js
#!/usr/bin/env node
const path = require('path');
const root = path.join('content-rating-display-213568', 'vod_content_rating_frontend');
const modules = ['app', 'list', 'utilities'].map(m => path.join(root, m));
console.log(JSON.stringify({
  android: {
    gradle_root: root,
    modules,
    instructions: [
      `cd ${root}`,
      './gradlew build'
    ]
  }
}, null, 2));
